sky/utils/schemas.py

```diff
@@ -1217,6 +1217,18 @@ def get_config_schema():
         },
     }
 
+    dashboard_schema = {
+        'type': 'object',
+        'required': [],
+        'additionalProperties': False,
+        'properties': {
+            'grafana_url': {
+                'type': 'string',
+                'pattern': r'^https?://.*$',
+            },
+        },
+    }
```

Comment:
When the helm chart sets `--set grafana.enabled=true`, we should probably autoconfigure this config/not need this config. Can be done as an env var.

Reply:
This is resolved in #5976.